akka-actor/src/main/scala/akka/pattern/AskSupport.scala

```diff
@@ -73,9 +192,9 @@ trait AskSupport {
    *
    */
   def ask(actorRef: ActorRef, message: Any)(implicit timeout: Timeout): Future[Any] =
-    actorRef.internalAsk(message, timeout, ActorRef.noSender)
```

Comment:
these changes should probably not be done: the old ask support was the most efficient solution for that problem, calling internalActorAsk means creating one closure per call that is then not really used (i.e. it only transports the final value anyway)